roles/generate-jenkins/templates/README_SNIPPETS/KASM.j2

```diff
@@ -0,0 +1,127 @@
+**Modern GUI desktop apps have issues with the latest Docker and syscall compatibility, you can use Docker with the `--security-opt seccomp=unconfined` setting to allow these syscalls on hosts with older Kernels or libseccomp**
+
+### Security
+
+**Do not put this on the Internet if you do not know what you are doing**
```

Comment:
```suggestion
{{ "Do not put this on the Internet if you do not know what you are doing." | admonition(flavour=markdown, severity="warning") }}
```
We can utilize github and mkdocs styling features by using this function. Also used here https://github.com/linuxserver/docker-jenkins-builder/blob/master/roles/generate-jenkins/templates/README_SNIPPETS/MEDIA.j2#L7